### ISSUE TITLE
Importing R2RML mappings into Protégé-ontop

### ISSUE THREAD

**Issue by PaulZH** [ontop/ontop#106]
- imported W3C's org and regorg vocabs
- established a datasource
- imported R2RML mapping
```
<ActivityMapping>
  a rr:TriplesMap;
  rr:logicalTable [ rr:tableName "activity" ];
  rr:subjectMap [
    rr:template "http://data.kbodata.be/organisation/{EntityNumber}#id";
    rr:class org:FormalOrganization;
    rr:termType rr:IRI 
  ];

 rr:predicateObjectMap [
    rr:predicate rov:orgActivity;
    rr:objectMap [
    rr:template "http://id.fedstats.be/nace{NaceVersion}/{NaceCode}#id";
    rr:termType rr:IRI
    ]
   ];
.
```
In the mapping manager, edit mapping.
I get
`<http://data.kbodata.be/organisation/{EntityNumber}#id> a <http://www.w3.org/ns/org#FormalOrganization> ; <http://www.w3.org/ns/regorg#orgActivity> <http://id.fedstats.be/nace{NaceVersion}/{NaceCode}#id> `
but surrounded in red with error message
```
`ERROR: The below list of predicates is unknown by the ontology:
- http://www.w3.org/ns/org#FormalOrganization
- http://www.w3.org/ns/regorg#orgActivity`
```

Querying via Quest although works.

**Comment by skomlaebri** [comment]
Hi, can you provide your ontology?
I just created a simple ontology, Lloaded your r2rml mapping in Protege and could not reproduce the problem. Here it's my ontology
<rdf:RDF xmlns="http://authority#"
     xml:base="http://authority"
     xmlns:schema="http://schema.org/"
     xmlns:owl="http://www.w3.org/2002/07/owl#"
     xmlns:org="http://www.w3.org/ns/org#"
     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
     xmlns:skos="http://www.w3.org/2004/02/skos/core#"
     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
     xmlns:terms="http://purl.org/dc/terms/"
     xmlns:xml="http://www.w3.org/XML/1998/namespace"
     xmlns:rov="http://www.w3.org/ns/regorg#"
     xmlns:foaf="http://xmlns.com/foaf/0.1/">
    <owl:Ontology rdf:about="http://authrity">
        <owl:imports rdf:resource="http://www.w3.org/ns/regorg"/>
        <owl:imports rdf:resource="http://www.w3.org/ns/org#"/>
    </owl:Ontology>
</rdf:RDF>

**Comment by PaulZH** [comment]
Thanks for coming back.
The 2 W3C vocabularies are attached.

Kind Regards,
Paul Hermans

-------------------------
ProXML bvba
Linked Data services
KBO: http://data.kbodata.be/organisation/0476_068_080#id
(w) www.proxml.be<http://www.proxml.be/>
(e) paul@proxml.be<mailto:paul@proxml.be>
(tw)  @PaulZH
(t)  +32 15 23 00 76
(m) +32 473 66 03 20


Narcisweg 17
3140 Keerbergen
Belgium


ODEdu – Innovative Open Data Education and Training based on PBL and Learning Analytics - http://odedu-project.eu/

OpenGovIntelligence  – Public Administration Modernization by exploiting Linked Open Statistical Data

OpenCube – Linked Open Statistical Data - http://opencube-project.eu/

**Comment by skomlaebri** [comment]
Hi, I didn't receive the attachments.
N.B. I modified my previous message to make my ontology readable.

**Comment by PaulZH** [comment]
Second trial. Attached.

Or downloadable from http://mbx.cm/t/xP0df

Paul

Kind Regards,
Paul Hermans

-------------------------
ProXML bvba
Linked Data services
KBO: http://data.kbodata.be/organisation/0476_068_080#id
(w) www.proxml.be<http://www.proxml.be/>
(e) paul@proxml.be<mailto:paul@proxml.be>
(tw)  @PaulZH
(t)  +32 15 23 00 76
(m) +32 473 66 03 20


Narcisweg 17
3140 Keerbergen
Belgium


ODEdu – Innovative Open Data Education and Training based on PBL and Learning Analytics - http://odedu-project.eu/

OpenGovIntelligence  – Public Administration Modernization by exploiting Linked Open Statistical Data

OpenCube – Linked Open Statistical Data - http://opencube-project.eu/

**Comment by skomlaebri** [comment]
Fixed in 5049cbc6c23e34a57776010af64eaef206569290. 
Add classes and properties of imported ontologies at runtime
